### ISSUE TITLE
Define Collection Output Template

### ISSUE THREAD

**Issue by drnek** [jazzhistorydatabase/jhdb-cms#114]
Mikel suggested the 2nd page template currently called 'collection' have the 'ability to link other collections and get a sidebar like we have on the landing page)'

let's draft a design layout and review. hand sketch is fine.

we should think about the content layout and assign this a more specific title. 

If there is a sidebar, will it have collection content or content from other collections on the site? 
the latter is a departure and we've thought about this, so if we want to implement it will need to be clearly non-collection material that is being shown to the user.   

the goal of a collection page should be to give an overview of the collection, a snapshot.  if you look at some older collections they are closer to what I picture but now with large media instead of small thumbs.. but an overview that is easy to digest.  ideally add the bio summary and bio photo on top of a preview. like our current artist pages but keep the content slim, add a description area..

example of old style with concise summaries and links to subpages http://www.jazzhistorydatabase.com/content/collections/Esposito_Jeremy/index.php   

let's get some new designs!

**Comment by matticoli** [comment]
@LucasNVarella you had something for this right?

**Comment by LucasNVarella** [comment]
Yep, though it's more of a medium-high fidelity prototype - it doesn't link to other collections yet, some text sizing is off, and it's not accessible without changing the server code directly at the moment. But it has the sidebar with fake links to other collections as we would want them, closely resembling the landing page for JHDB and Global.
![image](https://user-images.githubusercontent.com/10486753/83960060-65fbb200-a852-11ea-9555-7fd571388b76.png)
this is in a file called contributor.handlebars under server/views.
All we need after patching it up a bit is to link this up as an option in the UI and make sure the backend knows when to render it!

**Comment by matticoli** [comment]
#44 covers the latter part of that, but linking Pages together may be an entirely separate issue (and may be tied in some form to subpages? #111)

In any case @drnek Let us know your thoughts on the layout (with the assumption that the reported bugs/fixes for the existing template will be addressed on this one too)
